<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="nbairro">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Dados</string>
     </attribute>
     <widget class="QPushButton" name="Inserir">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Inserir</string>
      </property>
     </widget>
     <widget class="QLabel" name="nome">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Incidência Aeds Aegypiti</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>691</width>
        <height>211</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Zona</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bairro</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dengue</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Zika</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Chikungunya</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Incidência</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Risco</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="Limpar">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Números da Doença</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>430</y>
        <width>391</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="orbairro">
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Ordenar por bairro </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="orrisco">
         <property name="text">
          <string>Ordenar por risco</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>189</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="bairro">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Bairro:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bairro_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>182</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="zona">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Zona:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_zona">
         <item>
          <property name="text">
           <string>Norte</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sul</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Leste</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Oeste</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>122</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="degue">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Dengue:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ndengue"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="zika">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Zika:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nzika"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="chik">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Chikungunya:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nchick"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Estatística</string>
     </attribute>
     <widget class="QLabel" name="zmr">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>561</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Zona com maior Risco:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Doença com maior Incidência:</string>
      </property>
     </widget>
     <widget class="QLabel" name="dmi">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>561</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Doença com menor Incidência:</string>
      </property>
     </widget>
     <widget class="QLabel" name="dmie">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>561</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
